### ISSUE TITLE
French windows missing translations and popups missing language tagging (WCAG SC 11.2.4.4 - 11.3.1.1)

### ISSUE THREAD

**Issue by laurencechavet** [ThePacielloGroup/CCAe#329]
**Describe the bug**
French windows missing translations and popups missing language tagging

**Desktop (please complete the following information):**
 - OS: Windows 10 64 bits
 - Version CCA 3.4.2

**To Reproduce**
- Open the preferences, and choose the French language
- Open the French window “à propos”
-- the first link under “Liens externes” is not translated and appear empty.
-- the main paragraph is in English
- In the French window “Preferences” page
-- the last part “Copy results” is in English, including the buttons and text areas accessible names.
- In the French menu “edition”
-- the item “Copy short results” is in English
- In French window, “Colour blindness simulation”, “About”, and “Preferences””
-- The main page is properly tagged in English or French depending on the language selected. But the 3 popups “Simulateur de daltonisme”, “à propos”, and “Préférences” do not have a lang attribute and are not read with the proper pronunciation by text readers. (Only "About" shown in the screenshots, but the same for all the popups)

**Expected behavior**
All texts are translated
The French popups are read with a French pronunciation by text readers

**Screenshots**
![bug_11 2 4 4_MissingLinkText](https://github.com/ThePacielloGroup/CCAe/assets/65306416/42d26576-ebbe-4522-9f17-c5b699bf76bb)
![bug_11 3 1 1_FRAboutInEN](https://github.com/ThePacielloGroup/CCAe/assets/65306416/c4d5a134-24b0-4bf4-af87-c00962a84564)
![bug_11 3 1 1_FRPreferencesInEN](https://github.com/ThePacielloGroup/CCAe/assets/65306416/813224d5-70cf-4f70-8fad-e58f0587703a)
![bug_11 3 1 1_FRMenuInEN](https://github.com/ThePacielloGroup/CCAe/assets/65306416/b33b787a-1dcb-44ab-82a8-4de1b1d8c894)
![bug_11 3 1 1_FRPopupsLanguageNotDefined](https://github.com/ThePacielloGroup/CCAe/assets/65306416/43b113d6-0033-4444-9fc3-30fe43c61983)

**Additional context**
N/A

**Comment by ferllings** [comment]
Open the French window “à propos”
- [ ] the first link under “Liens externes” is not translated and appear empty.
- [ ] the main paragraph is in English
In the French window “Preferences” page
- [ ] the last part “Copy results” is in English, including the buttons and text areas accessible names.
In the French menu “edition”
- [ ] the item “Copy short results” is in English
In French window, “Colour blindness simulation”, “About”, and “Preferences””
- [ ] The main page is properly tagged in English or French depending on the language selected. But the 3 popups “Simulateur de daltonisme”, “à propos”, and “Préférences” do not have a lang attribute and are not read with the proper pronunciation by text readers. (Only "About" shown in the screenshots, but the same for all the popups)
